analyzer/src/main/kotlin/managers/utils/NuGetAllPackageData.kt

```diff
@@ -23,7 +23,7 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty
 
-internal class NuGetAllPackageData(
+class NuGetAllPackageData(
```

Comment:
commit msg: "public to that its nested" is unclear to me

commit: Why is it necessary to expose things beyond the module scope? E.g. why do the functions
which return `PackageSpec` need to be made public?

Reply:
> commit msg: "public to that its nested" is unclear to me

"to" was a typo and was supposed to read "so".

> Why is it necessary to expose things beyond the module scope?

I've solved this now in the alternative way and made `parseLicenses()` / `parseAuthors()` internal.